### DISCUSSION TITLE
Location of .vimrc for VSCode

### DISCUSSION THREAD

**Discussion by m0xai** [blinksh/blink#1581] [Support]
### Checklist

- [x] Using latest version on the [App Store](http://itunes.apple.com/app/id1156707581)
- [x] Read the [docs](http://blink.sh/docs/)
- [x] Searched for [existing GitHub issues](https://github.com/blinksh/blink/issues)

### Configuration
Blink Shell (AppStore Version) v15.0.7.536
iPadOS 15.4.1

### Describe the bug

I‘m trying to change keybindigs of Vim Plug-in from VSCode. In my computer i place .vimrc in $HOME but, I couldn’t figure out where I should locate it. 

I’m using Blink Shell with Github Codespaces. 

`/private/var/mobile/Containers/Shared/AppGroup/60A91B2B-7F08-4392-9537-902CD8A7E144/home` and `/private/var/mobile/Containers/Shared/AppGroup/60A91B2B-7F08-4392-9537-902CD8A7E144/home/.blink` doesn’t work.

**Comment by carloscabanero** [comment]
Our VS Code integration works with VSCode.net, which is a web only version. I think this would be a question for them or the Plug-In developers. When developing a plugin for the web, the .vimrc file may have to be placed at a different location that is accessible by the browser, as by default it won't be able to reach your home folder.

**Reply by m0xai** [reply]
Thank you for your reply. I find a workaround. I can keep some of my `.vimrc` settings in VSCode `settings.json` file. This may not a 100% replacement of a `.vimrc` but it satisfies my some keyboard shortcuts needs.
